### ISSUE TITLE
Using bigger gamemaker games

### ISSUE THREAD

**Issue by calcprogrammer** [KuromeSan/chovy-gm#9]
hey thanks for this project, its really awesome :)
i was wondering if the reason some bigger gamemaker 8.1 games crash on the psp is because the runner (Mr Karoshi) is a PSP minis. Is there some way to get out of "minis mode" (probably not a real thing) so that larger games can run
Thanks so much 👍

**Comment by KuromeSan** [comment]
"Minis mode" is set by a flag in param.sfo which I allready patched out

**Comment by calcprogrammer** [comment]
I see, so its probably that the PSP itself doesnt have enough RAM to run it...hmmm
That could also be why earlier i tried to work the Undertale Demo to run, but it just crashed the PSP
do you think if i decompiled the demo and changed some stuff it could run..?

**Comment by EGAMatsu** [comment]
> I see, so its probably that the PSP itself doesnt have enough RAM to run it...hmmm
> That could also be why earlier i tried to work the Undertale Demo to run, but it just crashed the PSP
> do you think if i decompiled the demo and changed some stuff it could run..?

doesn't the demo use studio 1.4?

**Comment by calcprogrammer** [comment]
> > I see, so its probably that the PSP itself doesnt have enough RAM to run it...hmmm
> > That could also be why earlier i tried to work the Undertale Demo to run, but it just crashed the PSP
> > do you think if i decompiled the demo and changed some stuff it could run..?
> 
> doesn't the demo use studio 1.4?

nope, the demo used gamemaker 8, however the full game was made in studio 1.4, there seems to be something in particular that crashes the psp when dealing with the demo tho, still trying to figure out what that is

**Comment by KuromeSan** [comment]
It prints error logs to strout so.

**Comment by EGAMatsu** [comment]
> > > I see, so its probably that the PSP itself doesnt have enough RAM to run it...hmmm
> > > That could also be why earlier i tried to work the Undertale Demo to run, but it just crashed the PSP
> > > do you think if i decompiled the demo and changed some stuff it could run..?
> > 
> > 
> > doesn't the demo use studio 1.4?
> 
> nope, the demo used gamemaker 8, however the full game was made in studio 1.4, there seems to be something in particular that crashes the psp when dealing with the demo tho, still trying to figure out what that is

Ahh, it loads external files.

**Comment by nkrapivin** [comment]
I tried to launch Undertale Demo long before you guys (in a chat with KuromeSan). I managed to get it to work but it has severe graphical glitches which make the game unplayable. So don't bother.

**Comment by calcprogrammer** [comment]
> I tried to launch Undertale Demo long before you guys (in a chat with KuromeSan). I managed to get it to work but it has severe graphical glitches which make the game unplayable. So don't bother.

Hey, almost a year late haha. How much progress did you make? Would it be possible to send me the file, I might try tinker around with it if you could let me know what you did to get it to launch atleast.
